### PR TITLE
connect DB

### PR DESCRIPTION
**Pull request by gkortsit**
Hi, 

- Initial database connection added and project folders setup. 
- You will need to run `yarn install` to add the new packages.
- I have create a temporary User table to pull data from and test the migrations.
- you ll need to run `yarn sequelize-cli db:migrate`
- possibly also `npx sequelize-cli db:seed:all`, sorry not sure
- to run the server with the database you will need to setup the .env file with the following variables:
```
DB_HOST=
DB_USER=
DB_PASS=
DB_NAME=
```
- after running `npm run dev` you should be able to see `Connection has been established successfully.` in the console

### CODE REVIEW

**Comment by gkortsit** [comment]
forgot to gitignore the .env file. I will send another PR later

**Comment by gkortsit** [comment]
Commited proposed changes @iredjee. 

Please review.

Thanks

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 node_modules
 package-lock.json
+.env
```
[the remark this answers is not in the document]
Comment (iredjee) [reply .gitignore]: Not fixed

File: server/.sequelizerc [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const path = require('path')
+
+module.exports = {
+  config: path.resolve('./config', 'index.js'),
+  'models-path': path.resolve('./database/models'),
+  'seeders-path': path.resolve('./database/seeders'),
+  'migrations-path': path.resolve('./database/migrations'),
+}
```
Comment (iredjee) [inline server/.sequelizerc]: New line is missing

File: server/config/prod.js [hunk 3]
```diff
@@ -1,3 +1,4 @@
 module.exports = {
   port: process.env.PORT || 3000,
+  production: process.env.DATABASE_URL,
```
Comment (iredjee) [inline server/config/prod.js]: Let's put db configs under `db` key.
```
db: {
  url: process.env.DB_URL,
},
```

File: server/config/test.js [hunk 4]
```diff
@@ -1,3 +1,9 @@
 module.exports = {
   port: 3000,
+  username: 'root',
```
Comment (iredjee) [inline server/config/test.js]: Possible to make some db url instead of multiple params?

File: server/config/dev.js [hunk 5]
```diff
@@ -1,3 +1,8 @@
 module.exports = {
   port: 3000,
+  username: process.env.DB_USER,
```
Comment (iredjee) [inline server/config/dev.js]: Could you change to be `db: { url }` ? And you use dotenv to load env variables, could you also please provide db url for development in the discord (direct message)

File: server/config/prod.js (hunk 3 above)
Comment (gkortsit) [reply server/config/prod.js]: There is no need to add a db key there. If you read the `index.js` file in the `models `folder all the db configuration happens in that file and then a `db` object is being exported.

**Pull request merged**